code-challenges.test.js

```diff
@@ -28,88 +28,98 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // a) Create a test with expect statements for each of the variables provided.
 describe("divisible", () => {
     it("decides if the number inside it is evenly divisible by three or not.", () => {
-        expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
-        expect(divisibleByThree()).toEqual("0 is divisible by three")
-        expect(divisibleByThree()).toEqual("-7 is not divisible by three")
-    })
-})
 
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+        expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
+
 
 // b) Create the function that makes the test pass.
-const divisibleByThree = (number) => {
-        if(number % 3 === 0){
-            return "15 is divisible by 3"
+const divisibleByThree = (num1) => {
+        if(num1.number % 3 === 0){
+            return `${num1.number} is divisibole by three`
+        } else if (num1.number % 3 !== 0 ){
+            return `${num1.number} is not divisible by three`
         }
-}
+    }
 
 // Pseudo code:
-//create a function called divisibleByThree that takes an object as an argument
-NEED HELP WITH ALL THESE 3 FUNCTIONS, I WILL BE ATEENDING OFFICE HOURS THIS WEDNESDAY
+//create a function divisible by three, takes paramaeter num1
+//use conditionals 
+//object - key value pair
+//use dot notation to acces value in the object
+//use modulo and sctrict equality operators 
+//return string interpolation with expected output
```

Comment:
Make sure you are writing your own pseudo code from your own understanding of the code